app/javascript/packs/containers/NavigationBar/DropdownMenuContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+class DropdownMenuContainer extends React.Component {
+
+    buildingChange(event) {
```

Comment:
In the spirit of documentation, either specify that `event.target.value` should be a building ID, or change your `onChange` function in line 15 to do something like:
```javascript
onChange={(e) => this.buildingChange(e.target.value))}
```
...so that you can name this argument `buildingId` instead!